cumulus/pallets/xcmp-queue/src/lib.rs

```diff
@@ -557,8 +557,11 @@ impl<T: Config> Pallet<T> {
 			new_page.extend_from_slice(&encoded_fragment[..]);
 			let last_page_size = new_page.len();
 			let number_of_pages = (channel_details.last_index - channel_details.first_index) as u32;
-			let bounded_page = BoundedVec::<u8, T::MaxPageSize>::try_from(new_page)
-				.map_err(|_| MessageSendError::TooBig)?;
+			let bounded_page =
+				BoundedVec::<u8, T::MaxPageSize>::try_from(new_page).map_err(|error| {
+					log::error!(target: LOG_TARGET, "Failed to create bounded message page: {error:?}");
```

Comment:
```suggestion
					log::debug!(target: LOG_TARGET, "Failed to create bounded message page: {error:?}");
```

same for all other `log::error!` - error logs are reserved for errors relevant to node operator - these types of runtime errors should be coming up only when debugging (otherwise validators get spammed with error logs for on-chain stuff that has no relevance to them).